package.json

```diff
@@ -6,34 +6,36 @@
     "dev": "next dev -p 5000",
     "debug": "NODE_OPTIONS='--inspect' next dev -p 5000",
     "build": "next build",
-    "prettier:fix": "npx prettier --write .",
-    "prettier:test": "npx prettier -c .",
+    "prettier:fix": "prettier --write .",
+    "prettier:test": "prettier -c .",
     "prestart": "next build",
-    "start": "next start -p $PORT",
+    "start": "next start -p 5001",
     "tests": "yarn test:unit && yarn test:e2e",
-    "test:e2e": "PORT=5001 NEXT_PUBLIC_ENV_NAME=test start-test 5001 cy:run",
-    "test:unit": "TZ=Europe/London node --expose-gc ./node_modules/.bin/jest --runInBand --logHeapUsage",
+    "test:e2e": "NODE_ENV=test start-test 5001 cy:run",
+    "test:unit": "jest --runInBand --logHeapUsage",
     "lint": "next lint",
-    "e2e:audit": "PORT=5001 NEXT_PUBLIC_ENV_NAME=test start-test 5001 cy:run:audit",
-    "e2e:interactive": "PORT=5001 NEXT_PUBLIC_ENV_NAME=test start-test 5001 cy:open",
-    "e2e:server": "PORT=5001 NEXT_PUBLIC_ENV_NAME=test start-test 5001",
-    "cy:run": "TZ=Europe/London cypress run --browser chrome --headless",
-    "cy:run:audit": "TZ=Europe/London cypress run --browser chrome --headless --env runAudit=true",
-    "cy:open": "TZ=Europe/London cypress open"
+    "e2e:audit": "NODE_ENV=test start-test 5001 cy:run:audit",
+    "e2e:interactive": "NODE_ENV=test start-test 5001 'cypress open'",
+    "e2e:server": "NODE_ENV=test start-test 5001",
```

Comment:
Started to have too many inline vars here so I switched to using `.env.test` etc